### PR TITLE
Add Optical Dampers CBM

### PR DESCRIPTION
**Pull request by Cyrano7**
Adds a new CBM that's bundled with the Anti-Glare Compensators. Installing the Anti-glare Compensators will give you both CBMs now. Its a lot like the enhanced hearing/sound dampeners bundle but for your eyes instead. Toggling the Optical Dampers blinds you. Works well with the sound dampeners and alarm system installed.

### CODE REVIEW

**Comment by RogueYun** [comment]
Kind of off topic but since you brought this up I had this question the other day. Is there a vision correction CBM out there for characters that are near/far sighted? Could this all be bundled together like transition lenses?

**Comment by Cyrano7** [comment]
@RogueYun Well the anti-glare bionic is more like the CBM equivalent to sunglasses, welding googles and now the blindfold. But I could see having a CBM equivalent to the binoculars that would fix the nearsighted/farsighted trait. Bundling them all together would be a bit overpowered I think. But good idea. Like an enhanced sight CBM.

**Comment by Zireael07** [comment]
I like both the dampers and the enhanced sight CBM.

**Comment by DeadLeaves** [comment]
Hello everyone my name is DeadLeaves and I'm a CBM addict. On a more serious note, I love everything CBM so the more options I have the better. Love this!

**Comment by Griffinhart** [comment]
The Diamond Cornea CBM seems like it would be a good candidate for also being a vision correction CBM.

File: data/json/items/bionics.json [hunk 1]
```diff
@@ -1333,7 +1333,7 @@
         "symbol": ":",
         "color": "yellow",
         "name": "Anti-Glare Compensators CBM",
-        "description": "Your eyes have high-power, quick-reacting transition lenses installed over them, negating any glare penalties and protecting you from bright flashes including those of a welder.",
+        "description": "Your eyes have high-power, quick-reacting transition lenses installed over them, negating any glare penalties and protecting you from bright flashes including those of a welder. Additionally you can toggle them to block out all light.",
```
Comment (illi-kun) [inline data/json/items/bionics.json]: Please use double spaces after periods.

File: data/json/bionics.json [hunk 2]
```diff
@@ -145,6 +145,12 @@
         "description": "Your eyes have quick-reacting transition lenses installed over them.  They negate glare penalties, partially protect you from bright flashes, and protect your eyes when welding."
     },{
         "type": "bionic",
+        "id": "bio_blindfold",
+        "name": "Optical Dampers",
+        "toggled": true,
+        "description": "The lenses installed over your eyes can be tinted to block out incoming light."
+	},{
```
Comment (illi-kun) [inline data/json/bionics.json]: This line contains tab instead of spaces. Please set up your text editor properly and don't use tabs.

**Comment by Cyrano7** [comment]
@RogueYun  @Zireael07  @DeadLeaves  @Griffinhart I wrote an idea for a enhanced vision CBM on the forums by the way if you wanted to add a comment. http://smf.cataclysmdda.com/index.php?topic=11748.15

**Pull request merged**